include/pokemon.h

```diff
@@ -313,14 +320,15 @@ struct SpeciesInfo /*0x24*/
  /* 0x05 */ u8 baseSpDefense;
  /* 0x06 */ u8 types[2];
  /* 0x08 */ u8 catchRate;
- /* 0x09 padding */
+ /* 0x09 */ u8 padding1;
  /* 0x0A */ u16 expYield; // expYield was changed from u8 to u16 for the new Exp System.
  /* 0x0C */ u16 evYield_HP:2;
             u16 evYield_Attack:2;
             u16 evYield_Defense:2;
             u16 evYield_Speed:2;
  /* 0x0D */ u16 evYield_SpAttack:2;
             u16 evYield_SpDefense:2;
+            u16 padding2:4;
  /* 0x0E */ u16 itemCommon;
  /* 0x10 */ u16 itemRare;
```

Comment:
Non-blocking: It might make sense to have a macro for items. If `itemCommon == itemRare` (as is the case for Lapras and Mystic Water) then the species _always_ holds that item.

Reply:
I think we could do this in the same vein of the typing automation we discussed where supplying the macro with a single argument makes it use it for both, but non-blocking indeed

Reply:
Maybe l'll do this in a future PR.